setup.cfg

```diff
@@ -55,7 +55,85 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
-
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+recommended =
```

Comment:
IMO `recommended` is rather vague. I would rename to  `performance` with `bottleneck, numexpr, numba`